x-pack/plugins/fleet/server/services/telemetry/fleet_usages_schema.ts

```diff
@@ -448,3 +448,51 @@ export const fleetUsagesSchema: RootSchema<any> = {
     _meta: { description: 'id of the deployment', optional: true },
   },
 };
+
+export const fleetIntegrationsSchema: RootSchema<any> = {
+  shared_integration_policies: {
+    type: 'long',
+    _meta: {
+      description: 'Count of total number of integration policies in this kibana',
```

Comment:
I think the description is mixed up for `shared_integration_policies` and `total_integration_policies`